src/main/java/Gui.java

```diff
@@ -0,0 +1,107 @@
+public class Gui {
+
+    /**
+     * Message displayed at startup of Duke.
+     */
+    public String printStartUp() {
+
+        return ("Hello, I'm Duke\nWhat can I do for you?\n\n");
+    }
+
+    /**
+     * Message displayed at termination of Duke.
+     */
+    public String terminateGui() {
+        return ("Bye. Hope to see you again soon!\n");
+    }
+
+    /**
+     * Message displayed when file cannot be read.
+     */
+    public String showLoadingError() {
+        return ("File could not be read!\n");
```

Comment:
not really showing the message, right? it is returning the message. so, maybe rename the method to something more appropriate?